youtube_dl_gui/locale/tr_TR/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,438 @@
+# Youtube-dlG localization file.
+# FIRST AUTHOR: Sotiris Papadopoulos <ytubedlg@gmail.com>, 2015.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: youtube-dlg 0.3.8\n"
+"POT-Creation-Date: 2015-07-24 04:31+EEST\n"
+"PO-Revision-Date: 2015-07-24 04:37+EEST\n"
+"Last-Translator: Ali KOÇ <alikoc@protonmail.ch>\n"
+"Language-Team: tr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+
+#: mainframe.py:82
+msgid "URLs"
+msgstr "Url(ler)"
+
+#: mainframe.py:83
+msgid "Download"
+msgstr "İndirme"
+
+#: mainframe.py:84
+msgid "Update"
+msgstr "Güncelleme"
+
+#: mainframe.py:85 optionsframe.py:40
+msgid "Options"
+msgstr "Ayarlar"
+
+#: mainframe.py:86
+msgid "Error"
+msgstr "Hata"
+
+#: mainframe.py:87
+msgid "Stop"
+msgstr "Dur"
+
+#: mainframe.py:88
+msgid "Info"
+msgstr "Bilgi"
+
+#: mainframe.py:89
+msgid "Welcome"
+msgstr "Hoşgeldiniz"
+
+#: mainframe.py:90
+msgid "Successfully downloaded {0} url(s) in {1} day(s) {2} hour(s) {3} minute(s) {4} second(s)"
+msgstr "{0} url(ler) {1} gün {2} saat {3} dakika {4} saniye içinde başarı ile indirildi"
+
+#: mainframe.py:92
+msgid "Downloads completed"
+msgstr "İndirmeler tamamlandı"
+
+#: mainframe.py:93
+msgid "Downloading {0} url(s)"
+msgstr "İnidiriliyor {0} url(ler)"
```

Comment:
Yes. "indiriliyor" is correct one. I'm on mobile right now. Could you please correct the line?